From: Hannes Reinecke <hare@suse.de>
Date: Wed, 15 Jan 2014 14:53:11 +0100
Subject: cciss: Set 'cciss_allow_hpsa' per default
Patch-Mainline: never
References: FATE#316683

For SLE12 cciss is superseded by hpsa, so enable 'cciss_allow_hpsa'
per default.

Signed-off-by: Hannes Reinecke <hare@suse.de>

diff --git a/drivers/block/cciss.c b/drivers/block/cciss.c
index edfa251..abc4217 100644
--- a/drivers/block/cciss.c
+++ b/drivers/block/cciss.c
@@ -75,7 +75,7 @@ module_param(cciss_simple_mode, int, S_IRUGO|S_IWUSR);
 MODULE_PARM_DESC(cciss_simple_mode,
 	"Use 'simple mode' rather than 'performant mode'");
 
-static int cciss_allow_hpsa;
+static int cciss_allow_hpsa = 1;
 module_param(cciss_allow_hpsa, int, S_IRUGO|S_IWUSR);
 MODULE_PARM_DESC(cciss_allow_hpsa,
 	"Prevent cciss driver from accessing hardware known to be "
